components/provisioner/assets/database/provisioner.sql

```diff
@@ -89,11 +89,17 @@ CREATE TABLE kyma_release
 
 -- Kyma Config
 
+CREATE TYPE kyma_profile AS ENUM (
+    'EVALUATION',
+    'PRODUCTION'
```

Comment:
Profile is defined within separate profile-{name}.yaml. Actually, there is no such thing as a default one. There are default settings - the ones specified in the chart that are used when no profile is set. When user wants to use profile they have two options reflected in the API: production and evaluation. Adding DEFAULT in the enum is not a good idea IMHO. Provisioner cannot do anything that makes sense with this value. If it would pass "default" string to the Installer, the Installer will look for "profile-default.yaml" and ignore profile settings as a result (please see the [code](https://github.com/kyma-project/kyma/blob/687c943ac9402dab2efe3cef9bae6785e8313e86/components/kyma-operator/pkg/kymahelm/kyma-helm-client.go#L350)). If it would pass empty string to Installer, the Installer will ignore the profile setting.

Reply:
Oh, that makes sense now! Thanks :D